### PR TITLE
[Android][Gui] Display disk usage on project tab

### PR DESCRIPTION
**Pull request by talregev**
Display disk usage on project tab:

Screenshot_20220122-235924_BOINC

### CODE REVIEW

File: android/BOINC/app/src/main/java/edu/berkeley/boinc/adapter/ProjectsListAdapter.java [hunk 1]
```diff
@@ -287,6 +294,10 @@ else if(trans.getNextRequestTime() < nextRetryS || nextRetryS == 0) {
                                                                        NumberFormat.getIntegerInstance().format(hostCredit) :
                                                                        this.activity.getString(R.string.projects_credits_host_and_user, hostCredit, userCredit));
 
+            TextView tvDiskUsage = vi.findViewById(R.id.project_disk_usage);
```
Comment (AenBleidd) [inline android/BOINC/app/src/main/java/edu/berkeley/boinc/adapter/ProjectsListAdapter.java:297]: Please fix indents

**Review by AenBleidd — CHANGES_REQUESTED** [review]

File: android/BOINC/app/src/main/java/edu/berkeley/boinc/adapter/ProjectsListAdapter.java (hunk 1 above)
Comment (talregev) [reply android/BOINC/app/src/main/java/edu/berkeley/boinc/adapter/ProjectsListAdapter.java:297]: This is the correct indent.
Please open full file view (or open more up and down) to see that it correct indent.

File: android/BOINC/app/src/main/res/layout/projects_layout_listitem.xml [hunk 2]
```diff
@@ -110,6 +110,31 @@
                     android:text="@string/projects_credits_host_and_user"
                     android:textAppearance="?android:attr/textAppearanceSmall"
                     android:textColor="?android:attr/textColorPrimary"/>
+            </LinearLayout>
```
Comment (AenBleidd) [inline android/BOINC/app/src/main/res/layout/projects_layout_listitem.xml:113]: Please fix indents

File: android/BOINC/app/src/main/res/layout/projects_layout_listitem.xml [hunk 3]
```diff
@@ -110,6 +110,31 @@
...
+
+            <LinearLayout
```
Comment (AenBleidd) [inline android/BOINC/app/src/main/res/layout/projects_layout_listitem.xml:115]: Please fix indents

**Review by AenBleidd — CHANGES_REQUESTED** [review]

File: android/BOINC/app/src/main/res/layout/projects_layout_listitem.xml (hunk 3 above)
Comment (talregev) [reply android/BOINC/app/src/main/res/layout/projects_layout_listitem.xml:115]: Please look at whole file. This is the correct indent

File: android/BOINC/app/src/main/res/layout/projects_layout_listitem.xml (hunk 2 above)
Comment (talregev) [reply android/BOINC/app/src/main/res/layout/projects_layout_listitem.xml:113]: Please look at whole file. This is the correct indent

File: android/BOINC/app/src/main/res/layout/projects_layout_listitem.xml (hunk 3 above)
Comment (AenBleidd) [reply android/BOINC/app/src/main/res/layout/projects_layout_listitem.xml:115]: It's still not correct:
image

Comment (talregev) [reply android/BOINC/app/src/main/res/layout/projects_layout_listitem.xml:115]: ok. I will fix it. Thank you for noticing

**Review by AenBleidd — APPROVED** [review]

**Pull request merged**